lesson_08/homeworks.py

```diff
@@ -0,0 +1,34 @@
+
+def trees_total(apple, peer, plum):
+    '''Функція приймає кількість яблукб груш і слив, та виводить загальну кількусть дерев у саду'''

```

Comment:
```suggestion
    '''Функція приймає кількість яблукб груш і слив, та виводить загальну кількусть дерев у саду'''

```
це щось ну дуже проста функція